NightscoutServiceKit/RemoteCommands/V1/Notifications/CarbRemoteNotification.swift

```diff
@@ -55,4 +55,42 @@ public struct CarbRemoteNotification: RemoteNotification, Codable {
     public static func includedInNotification(_ notification: [String: Any]) -> Bool {
         return notification["carbs-entry"] != nil
     }
+    
```

Comment:
I thought it may be best to adjust the start-time as soon as we receive the notification, rather than further down the line (like when we upload). That way Loop is consistent in its view of this date field -- it doesn't change later during the process leading to broken assumption.

Reply:
I added some comments to the NS ticket. This is, at its heart, a problem in NS.

I agree that any time hacks like this should be done as early in the chain as possible. But this particular change seems to be mid-chain. Can't we be adding in seconds/or microseconds on the caregiver side? It would be better to fix it at user input time, by using the seconds field of the *actual* created at time (i.e. the time of user entry), like currently happens for carbs entered via the Loop UI.